ingestion/src/metadata/cmd.py

```diff
@@ -123,4 +131,80 @@ def report(config: str) -> None:
         ) from exc
 
 
+@metadata.command()
+@click.option(
+    "-t",
+    "--type",
+    help="Workflow config",
+    required=True,
+)
+@click.option(
+    "-p",
+    "--path",
```

Comment:
we should just accept 2 options --run -r and --clear -c. 
1. run should download the docker-compose.yaml from git repo and check the requirements and run the docker
2. clear should clean up all the docker with our label and clear the network etc..